exercises/tournament/tournament_test.rb

```diff
@@ -5,65 +5,64 @@
 
 # Common test data version: cda8f98
 class TournamentTest < Minitest::Test
-
   def test_typical_input
     # skip
     input = <<-INPUT.gsub(/^ */, '')
-      Allegoric Alaskans;Blithering Badgers;win
-      Devastating Donkeys;Courageous Californians;draw
-      Devastating Donkeys;Allegoric Alaskans;win
-      Courageous Californians;Blithering Badgers;loss
-      Blithering Badgers;Devastating Donkeys;loss
-      Allegoric Alaskans;Courageous Californians;win
-    INPUT
+Allegoric Alaskans;Blithering Badgers;win
```

Comment:
Why should these lines be indented? 

Yes, they were indented in the old version, but ...

We don't indent the heredoc for beer-song, why are we indenting these lines?

Looking at the README, there's no inherent reason for the lines in the heredoc to be indented. The only reason it is indented is to `indent` the test. We wouldn't expect a real problem to change the way its data was configured just so the test appeared correctly indented!